pynest/nest/lib/hl_api_nodes.py

```diff
@@ -83,12 +83,13 @@ def Create(model, n=1, params=None, positions=None):
     model_deprecation_warning(model)
 
     if positions is not None:
+        spatial = getattr(nest.NestModule, "spatial")
```

Comment:
I'm surprised this works, I'd expect this to return `<property object 'spatial' at 0x...>`

Reply:
This was one of the more desperate tries and I was also surprised. If you find something better, go ahead, but I'm also fine with this. Maybe it should get a comment if it stays...

Reply:
Fine with it, You could maybe link to #2134 in the comment.